birdhouse/components/raven/default.env

```diff
@@ -1,10 +1,14 @@
 # The Geoserver that Raven will connect to.
 # Same default value as
-# https://github.com/CSHS-CWRA/RavenPy/blob/2e56041b605e83ab28ffdc5d817e645481dcc5fc/ravenpy/utilities/geoserver.py#L51
+# https://github.com/CSHS-CWRA/RavenPy/blob/1dc534cd3b7faed2ff18a90a915d448b2fca6bf7/src/ravenpy/utilities/geoserver.py#L50-L52
+# https://github.com/Ouranosinc/raven/blob/dc5a3750c551c3d2aa5b5a8c9d61190ed4f4c3ba/src/raven/utilities/geoserver.py#L43-L45
 # This is the production Geoserver that is always available with appropriate data.
 # For site that want to run your own Geoserver with your own data, please
 # override this variable with your own Geoserver instance.
 # Ex: RAVEN_GEO_URL="${BIRDHOUSE_PROXY_SCHEME}://${BIRDHOUSE_FQDN}/geoserver/"
+# The mapping to GEO_URL, RAVEN_GEOSERVER_URL, RAVENPY_GEOSERVER_URL
+# are done in components/raven/docker-compose-extra.yml.
+# RAVEN_GEO_URL will be our "stable" interface in birdhouse-deploy.
 __DEFAULT__RAVEN_GEO_URL="https://pavics.ouranos.ca/geoserver/"
 export RAVEN_GEO_URL='${__DEFAULT__RAVEN_GEO_URL}'
```

Comment:
Shouldn't this default be `${BIRDHOUSE_FQDN_PUBLIC}/geoserver/` and have a component dependency to `geoserver`? 

What is the interaction between `raven` and `pavics` when this component is enabled on another instance?

Reply:
Ok. Sounds good.